test/index.test.js

```diff
@@ -2,10 +2,12 @@
 
 const Web3 = require('web3')
 const web3 = new Web3('http://localhost:8545')
+const qtumrpc = 'http://qtum:test@ec2-54-172-219-152.compute-1.amazonaws.com:3889'
```

Comment:
Let's not expose internal testing URLs. All defaults in the code should be set to work 100% in the local dev env (i.e. `localhost`).

Reply:
done